db/migrate/20171010151218_import_legacy_specialist_publisher_first_published_at_dates.rb

```diff
@@ -0,0 +1,29 @@
+class ImportLegacySpecialistPublisherFirstPublishedAtDates < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  # ~4832 records match
+  def up
+    count = 0
+    content_ids = []
+    csv_data = CSV.read(Rails.root.join("db", "migrate", "data", "specialist-publisher-legacy-first-published-at.csv"))
+    csv_data.each do |content_id, slug, first_published_at|
+      updated = Edition.joins(:document)
+                       .where(publishing_app: "specialist-publisher",
+                              state: %w(draft published unpublished),
```

Comment:
can we also do this for superseded records since they share the same first_published_at data? (and we really want to rid this rod)